atd-vze/src/Components/TableSearchBar.js

```diff
@@ -20,6 +21,7 @@ const TableSearchBar = ({ setSearchFilter, clearFilters, fieldsToSearch }) => {
   const [searchValue, setSearchValue] = useState("");
   const [isDropdownOpen, setIsDropdownOpen] = useState(false);
   const [fieldToSearch, setFieldToSearch] = useState("");
```

Comment:
I think we need a default value for the `fieldToSearch` dropdown.

Reply:
I modified the code in the PR around `""` being the default value for the field. The form will never submit with this value set in `fieldToSearch` and the popup warning depends on this value being set in state to prevent the app from crashing.